src/architect/secret/secret.utils.ts

```diff
@@ -32,7 +33,8 @@ export default class SecretUtils {
 
   static async batchUpdateSecrets(app: AppService, secrets: Secret[], account: Account, options?: SecretOptions): Promise<void> {
     if (options?.environment_name) {
-      const environment = await EnvironmentUtils.getEnvironment(app.api, account, options?.environment_name);
+      const getEnvironmentOptions: GetEnvironmentOptions = { environment_name: options.environment_name };
```

Comment:
Thanks for adding this options object. Can you make the variable namec `getEnvironmentOptions` snake case where it's used please? To be consistent

Reply:
updated references to use snake case;